Makefile.calico

```diff
@@ -78,6 +81,11 @@ image-all: $(addprefix sub-image-,$(VALIDARCHES))
 sub-image-%:
 	$(MYMAKE) image ARCH=$*
 
+.PHONY: dind-image
+dind-image: $(DIND_CONTAINER_NAME)
+$(DIND_CONTAINER_NAME): $(BINARIES)
+	docker build -t $(DIND_CONTAINER_NAME):latest-$(ARCH) -f dind-image/Dockerfile.$(ARCH) dist/$(ARCH)
```

Comment:
Inside the rule you can use the automatic variable `$@` to refer to the target of the rule
```suggestion
	docker build -t $@:latest-$(ARCH) -f dind-image/Dockerfile.$(ARCH) dist/$(ARCH)
```